src/antsibull/data/ansible-forum-announcement.j2

```diff
@@ -8,6 +8,9 @@
 Hello everyone,
 
 We're happy to announce the release of the Ansible {{ version }} package{% if is_prerelease %} pre-release{% endif %}!
+{% if end_of_life %}
+This is the final {{ major_version }}.X.X release.
```

Comment:
Is it worth bolding either this or the message at the end? The forum announcements are all markdown now that the MLs are shut down.

Reply:
I think making this one bold is a good idea.

Reply:
92c3723bbe7d2c0c85d7415b9bee57d045906661